rama-cli/src/cmd/fp/data.rs

```diff
@@ -306,50 +306,52 @@ pub(super) fn get_tls_display_info(ctx: &Context<Arc<State>>) -> Option<TlsDispl
             .map(|extension| match extension {
                 ClientHelloExtension::ServerName(domain) => TlsDisplayInfoExtension {
                     id: extension.id().to_string(),
-                    data: TlsDisplayInfoExtensionData::Single(match domain {
-                        Some(domain) => domain.to_string(),
-                        None => "".to_owned(),
-                    }),
+                    data: match domain {
```

Comment:
In cases like this better to use something like `map` as it:

- makes the intention of your code immediately clear
- less room for errors
- less boilerplate

```rust
data: domain.map(|d| TlsDisplayInfoExtensionData::Single(d.to_string()),
```